src/modules/users/components/UsersList/UsersList.tsx

```diff
@@ -16,11 +16,11 @@ import Filtration from "../../../shared/components/Filtration/Filtration";
 import useFetch from "../../../../hooks/useFetch";
 import Pagination from "../../../shared/components/Pagination/Pagination";
 import ViewDetailsModal from "../../../shared/components/ViewDetailsModal/ViewDetailsModal";
+import UpDownArrows from "../../../shared/components/SvgIcons/SvgIcons";
 
 const UsersList = () => {
   const [pageNum, setPageNum] = useSearchParams();
 
-  
   const [usersList, setUsersList] = useState<UsersListResponse[]>([]);
```

Comment:
We can store one state (the users api response) and derive the other states from it instead of having three different states.